git/attribs.go

```diff
@@ -64,19 +67,36 @@ func GetRootAttributePaths(mp *gitattr.MacroProcessor, cfg Env) []AttributePath
 // GetSystemAttributePaths behaves as GetAttributePaths, and loads information
 // only from the system gitattributes file, respecting the $PREFIX environment
 // variable.
-func GetSystemAttributePaths(mp *gitattr.MacroProcessor, env Env) []AttributePath {
-	prefix, _ := env.Get("PREFIX")
-	if len(prefix) == 0 {
-		prefix = string(filepath.Separator)
-	}
+func GetSystemAttributePaths(mp *gitattr.MacroProcessor, env Env) ([]AttributePath, error) {
+	var path string
+	if IsGitVersionAtLeast("2.42.0") {
+		cmd, err := gitNoLFS("var", "GIT_ATTR_SYSTEM")
+		if err != nil {
+			return nil, errors.New(tr.Tr.Get("failed to set up `git var GIT_ATTR_SYSTEM`: %v", err))
```

Comment:
As a minor note, it looks like [many](https://github.com/git-lfs/git-lfs/blob/1820b646e05c24497652b3f9eaf91500dc1ee988/git/git.go#L392) of our [other](https://github.com/git-lfs/git-lfs/blob/1820b646e05c24497652b3f9eaf91500dc1ee988/git/git.go#L261) [invocations](https://github.com/git-lfs/git-lfs/blob/1820b646e05c24497652b3f9eaf91500dc1ee988/git/git.go#L614) of `gitNoLFS()` return any error with the message `failed to find` rather than `failed to set up`, so I might suggest doing the same here, just for consistency.

Reply:
Good point.  I've updated both.